Lsunae/ElevenStreetSample/app/src/main/java/com/exam/elevenstreet/data/source/local/ProductLocalDataSourceImpl.kt

```diff
@@ -0,0 +1,24 @@
+package com.exam.elevenstreet.data.source.local
+
+import com.exam.elevenstreet.data.ProductCallback
+import com.exam.elevenstreet.data.parser.ProductXmlPullParserHandler
+import com.exam.elevenstreet.view.product.App
+
+class ProductLocalDataSourceImpl : ProductLocalDataSource {
+
+    val context = App.instance.context()
```

Comment:
```suggestion
    private val context = App.instance.context()
```
외부에서 접근할 필요는 없어보입니다.

Reply:
수정했습니다!
0b02636